ui/apps/platform/cypress/helpers/formHelpers.js

```diff
@@ -5,7 +5,7 @@ export function getInputByLabel(label) {
         .contains('label', label)
         .invoke('attr', 'for')
         .then((id) => {
-            cy.get(`#${CSS.escape(id)}`);
+            cy.get(`#${CSS.escape(id)}`).scrollIntoView();
```

Comment:
As a follow after successful upgrade, we might be able to replace these imperative helpers with ordinary selectors which have `:has` for the label element.